data/json/bionics.json

```diff
@@ -472,22 +467,22 @@
     "type": "bionic",
     "name": { "str": "Fingerhack" },
     "description": "One of your fingers has an electrohack surgically embedded in it; an all-purpose hacking unit used to override control panels and the like (but not computers).  Skill in computers is important, and a failed use may damage your circuits.",
-    "occupied_bodyparts": [ [ "hand_r", 2 ] ]
+    "occupied_bodyparts": [ [ "hand_l", 1 ] ]
```

Comment:
again why going from right to left?

Reply:
The only dexterity this would appear to need is enough to insert it into the right slot.

Reply:
Butthack? Nosehack?
Torso implant with a short cable included?

Reply:
Possible on the last. Having a cable coming from, to observers, your clothing would look a lot less suspicious than using your finger, since there aren't interface ports in CDDA.

Reply:
Changed to "deployable hacking unit in arm", two slots in right arm.